src/transactionsFactories/transferTransactionsFactory.ts

```diff
@@ -82,18 +82,10 @@ export class TransferTransactionsFactory {
         return this.gasEstimator !== undefined;
     }
 
-    private ensureMembersAreDefined() {
+    private ensureConfigIsDefined() {
```

Comment:
Indeed, the presence of `this.config` indicates the presence of `tokenComputer` and `tokenTransfersDataBuilder`, as well. Also, we can set `this.tokenComputer` and `this.tokenTransfersDataBuilder` in constructor no matter what (irrespective to `isGasEstimator()`).

In v14, this is the first legacy thing to be dropped :)